include/mbedtls/ecp.h

```diff
@@ -53,139 +65,151 @@ extern "C" {
 #endif
 
 /**
- * Domain parameters (curve, subgroup and generator) identifiers.
+ * Domain parameters: curve, subgroup, and generator.
```

Comment:
I preferred the original wording including `identifiers` - the enum elements _represent_ domain parameters, but aren't such themselves.